<?xml version="1.0"?>
<launch>
    <param name="max_planning_time"     type="double" value="20.0" />
    <param name="num_planning_attempts" type="int"    value="10" />
    
    <node pkg="ra1_pro_move_cartesian" type="ra1_pro_move_cartesian_node" name="ra1_pro_move_cartesian_node" args="" output="screen"/>
</launch>
